fvtest/compilertest/CMakeLists.txt

```diff
@@ -54,7 +54,6 @@ elseif(OMR_ARCH_S390)
 endif()
 
 if(OMR_ARCH_X86 OR OMR_ARCH_S390)
-	set(OMR_WARNINGS_AS_ERRORS ON)
```

Comment:
Unless I'm mistaken, this is the means of enabling warnings-as-errors for the compiler component on x86 and Z.  By removing this line I believe we'll be permitting warnings again on those platforms.

Should this code instead check whether the option was already defined on the command-line and only if it has not been then set OMR_WARNINGS_AS_ERRORS on?  I am unsure of the exact CMake syntax to do a check for that, but think that should be possible.  What do you think?

Reply:
`config.cmake` defines the default to be `ON`, so if a user has not explicitly disabled warnings-as-errors, line 57 would simply be redundant. On the other hand if treating warnings-as-errors is unwanted, this would not cooperate.

Reply:
@0xdaryl Any other concerns?